### ISSUE TITLE
Landing Guidance Autopilot has issue with Time warp

### ISSUE THREAD

**Issue by Nio9345** [MuMech/MechJeb2#950]
There's seems to be a problem with the time-warp when using MechJeb's Landing Guidance. When time-warp is used the vessel just "snaps" away from its trajectory and when time-warp stops the trajectory is way off. I used the latest stable build and build #751.

**Comment by Xteuk** [comment]
I've had this happening too, but outside of any MechJeb's.
It happens sometimes after using HyperEdit to pop a vessel into orbit.
Not sure if it relates to your issue, though.

**Comment by Nio9345** [comment]
I have read articles here having the same problem. I removed Huper edit and I'm not using any planet mods. I also reinstalled everything from CKAN and deleted old files. I still have the same problem.

**Comment by Nio9345** [comment]
![20171129150935_1](https://user-images.githubusercontent.com/5494353/33376728-7cf3cee0-d517-11e7-99e0-3340d9e0a691.jpg)

This is also a screenshot to show what happens. I use KRASH so I can put it in orbit fast so I can test it. I used it also while I originally posted this problem.

**Comment by sarbian** [comment]
I do not think this has anything to do with MJ since it does not edit the orbits or add forces. It s either a stock problem or related to an other mod. Mods like Krash are always highly suspect to me since moving a vessel by code is quite error prone when you don't know exactly how stuff works (and it was hard enough even when knowing it).

**Comment by BIGMOS** [comment]
I am using the latest version an it still does not work. with time warp

**Comment by Gordon-Dry** [comment]
Correct, I also had this last session.
#801

**Comment by lamont-granquist** [comment]
I've had Kerbal Alarm Clock mess up orbits when I pressed the "warp to SOI" button in the map -- although I suspect its not KAC messing up the orbit, but it messed up the orbit when I press the KAC button.  I'd suspect a mod like PersistentRotation or something that otherwise hacked up the motion of the vessel rather than MJ.

> I do not think this has anything to do with MJ since it does not edit the orbits or add forces.

+1 on this.  Even if it is a problem with MJ I couldn't guess in the code where this was happening, since nothing in the MJ codebase should be doing this.

**Comment by lamont-granquist** [comment]
This isn't mechjeb, this is stock bugginess.
